frontend/src/metabase/query_builder/components/view/View/NotebookContainer/NotebookContainer.tsx

```diff
@@ -64,31 +64,35 @@ export const NotebookContainer = ({
 
   const transformStyle = isOpen ? "translateY(0)" : "translateY(-100%)";
 
-  const Handle = forwardRef<HTMLDivElement, Partial<ResizableBoxProps>>(
-    function Handle(props, ref) {
-      const handleWidth = 10;
-      const borderWidth = 1;
-      const left = rem(-((handleWidth + borderWidth) / 2));
-
-      return (
-        <Box
-          data-testid="notebook-native-preview-resize-handle"
-          ref={ref}
-          {...props}
-          pos="absolute"
-          top={0}
-          bottom={0}
-          m="auto 0"
-          w={rem(handleWidth)}
-          left={left}
-          style={{
-            zIndex: 5,
-            cursor: "ew-resize",
-          }}
-        ></Box>
-      );
-    },
-  );
+  const Handle = forwardRef<
+    HTMLDivElement,
+    Partial<ResizableBoxProps> & { onResize?: any } //Mantine and react-resizeable have different opinions on what onResize should be
+  >(function Handle(props, ref) {
+    const handleWidth = 6;
```

Comment:
did we mean to change the size from 10 -> 6?

Reply:
on second look it appears these changes might have slipped in / are unrelated to the PR

Reply:
I have no idea how that got there haha